distribution/test-release.sh

```
@@ -4,15 +4,15 @@
 set -e
 
 VERSION="${1:-1.0.0}"
-PACKAGE_NAME="perl-lsp"
+PACKAGE_NAME="perllsp"
```

Shellcheck (0.11.0)

[warning] PACKAGE_NAME appears unused. Verify use (or export if used externally).

(SC2034)